gslib/utils/copy_helper.py

```diff
@@ -1435,21 +1435,20 @@ def ExpandUrlToSingleBlr(url_str,
   if storage_url.IsBucket():
     return (storage_url, True)
 
-  # For object/prefix URLs, there are four cases that indicate the destination
+  # For object/prefix URLs, there are three cases that indicate the destination
   # is a cloud subdirectory; these are always considered to be an existing
   # container. Checking each case allows gsutil to provide Unix-like
   # destination folder semantics, but requires up to three HTTP calls, noted
   # below.
 
-  # Case 1: If a placeholder object ending with '/' exists.
-  if IsCloudSubdirPlaceholder(storage_url):
```

Comment:
You can now remove this import as it's unused.

Reply:
Done